index.html

```diff
@@ -97,200 +104,325 @@ <h2 id="connect">Let's Connect</h2>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                </li>
+                <li class="btn-container">
+                  <button type="button" class="btn">
+                    <a href="#">See Project</a>
+                  </button>
+                </li>
+              </ul>
+            </div>
+          </article>
         </li>
         <li class="work-card">
-          <img
-            src="images/work_3.jpg"
-            alt="work_tonic_daily_personalized_reads"
-          />
-          <h2>Tonic</h2>
-          <ul class="experience">
-            <li>Canopy</li>
-            <li>&#x2022;</li>
-            <li>Back End Dev</li>
-            <li>&#x2022;</li>
-            <li>2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="technologies">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-          </ul>
-          <div id="button-container">
-            <button type="button" class="btn">
-              <a href="#">See Project</a>
-            </button>
-          </div>
+          <article class="work-card__article">
+            <div class="work-card__header">
+              <picture>
+                <source
+                  srcset="images/work_3_lg.png"
+                  media="(min-width: 768px)"
+                />
+                <img
+                  src="images/work_3.jpg"
+                  alt="work_tonic_daily_personalized_reads"
+                />
+              </picture>
+            </div>
+            <div class="work-card__body">
+              <ul>
+                <li class="work-card__title mobile">Tonic</li>
+                <li class="work-card__title desktop">Facebook 360</li>
+                <li>
+                  <ul class="work-card__experience mobile">
+                    <li>Canopy</li>
+                    <li>&#x2022;</li>
+                    <li>Back End Dev</li>
+                    <li>&#x2022;</li>
+                    <li>2015</li>
+                  </ul>
+                  <ul class="work-card__experience desktop">
+                    <li>FACEBOOK</li>
+                    <li>&#x2022;</li>
+                    <li>Full Stack Dev</li>
+                    <li>&#x2022;</li>
+                    <li>2015</li>
+                  </ul>
+                </li>
+                <li class="work-card__description mobile">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </li>
+                <li class="work-card__description desktop">
+                  Exploring the future of media in Facebook's first Virtual
+                  Reality app; a place to discover and enjoy 360 photos and
+                  videos on Gear VR.
+                </li>
```

Comment:
- [ ] [OPTIONAL but highly recommended] I would suggest you kindly make this part of all the project cards to be inside of a `p` tag rather than having it inside of a `li` tag 😄 😄 😄 , and this way will be semantically correct and the screen readers and other technologies will thank you